modules/dna_rna_tools.py

```diff
@@ -0,0 +1,196 @@
+COMPLEMENT_DNA = {'a': 't', 'A': 'T', 't': 'a', 'T': 'A',
+                  'g': 'c', 'G': 'C', 'c': 'g', 'C': 'G'}
+
+COMPLEMENT_RNA = {'a': 'u', 'A': 'U', 'u': 'a', 'U': 'A',
+                  'g': 'c', 'G': 'C', 'c': 'g', 'C': 'G'}
+
+
+def check_dna(sequence: str) -> bool:
+    """
+    Checkes is input string a DNA sequence.
+    If input sequence is DNA it will return True.
+    
+    Input:
+    -sequence (str): input nucleic acid sequence, any letter case is suitable.
+    
+    Output:
+    returns True or False (bool), is nucleic acid is DNA.
+    """
+    flag = True
+    valid_nucleotides_dna = {'A', 'T', 'C', 'G'}
+    unique_chars = set(sequence.upper())
+    if not unique_chars.issubset(valid_nucleotides_dna):
+        flag = False
+    return flag
```

Comment:
Зачем тут этот флаг?
```suggestion
    valid_nucleotides_dna = {'A', 'T', 'C', 'G'}
    unique_chars = set(sequence.upper())
    return unique_chars.issubset(valid_nucleotides_dna):
```